README.md

```diff
@@ -1,35 +1,59 @@
-# :fishing_pole_and_fish: config [![GoDoc][doc-img]][doc] [![GitHub release][release-img]][release] [![Build Status][ci-img]][ci] [![Coverage Status][cov-img]][cov] [![Report Card][report-card-img]][report-card]
+# :fishing_pole_and_fish: config [![GoDoc][doc-img]][doc] [![Build Status][ci-img]][ci] [![Coverage Status][cov-img]][cov]
 
-Package `config` allows users to:
-
-* Get components working with minimal configuration
-* Override any field if the default doesn't make sense for their use case
+Convenient, injection-friendly YAML configuration.
 
 ## Installation
-We recommend locking to [SemVer](http://semver.org/) range `^1` using
-[Glide](https://github.com/Masterminds/glide):
 
 ```
-glide get 'go.uber.org/config#^1
+go get -u go.uber.org/config
 ```
 
-## Stability
+Note that config only supports the two most recent minor versions of Go.
+
+## Quick Start
+
+```
+// Model your application's configuration using a Go struct.
+type cfg struct {
+    Parameter string
+}
+
+// Two sources of configuration to merge.
+base := strings.NewReader("module: {parameter: foo}")
+override := strings.NewReader("module: {parameter: bar}")
+
+// Merge the two sources into a Provider. Later sources are higher-priority.
+// See the top-level package documentation for details on the merging logic.
+provider, err := config.NewYAML(config.Source(base), config.Source(override))
+if err != nil {
+    panic(err) // don't do this in production
+}
+
+var c cfg
+if err := provider.Get("module").Populate(&c); err != nil {
+  panic(err) // don't do this in production
+}
+
+fmt.Printf("%+v\n", c)
+// Output:
+// {Parameter:bar}
+```
 
-This library is `v1` and follows [SemVer](http://semver.org/) strictly.
+## Development Status: Stable
 
-No breaking changes will be made to exported APIs before `v2.0.0`.
+All APIs are finalized, and no breaking changes will be made in the 1.x series
+of releases. Users of semver-aware dependency management systems should pin
+config to `^1`.
+
+<hr>
```

Comment:
Replace this line with `---` to use Markdown to do the same thing.

Reply:
o nift-ay